### ISSUE TITLE
[FR] Option to ignore 'The' and other articles

### ISSUE THREAD

**Issue by ebivan** [gateship-one/odyssey#75]
Hey there,
I've been using Odyssey for a couple of months now and I really love it!

But of course there are some things I'm missing.
I would love to have an option to ignore articles in the artist listing (especially 'THE' but it should also allow to define a list of ignored first words so that people could set it according to their needs. In my case that would be 'the,der, die,das,le,la'

Keep up the good work!

**Comment by djselbeck** [comment]
This is quite language specific. I'm not sure what the best way to implement something like this. 

Might be worth checking some other popular applications like kodi.

**Comment by ebivan** [comment]
Yes, Kodi has exactly this. It has a setting to ignore 'the' and allows to define a comma separated list right next to that setting of other first words to be ignored.
